fastlane_bot/helpers/txhelpers.py

```diff
@@ -66,58 +57,6 @@ def __post_init__(self):
         self.wallet_address = str(self.local_account.address)
 
         self.alchemy_api_url = self.ConfigObj.RPC_URL
-        self.nonce = self.get_nonce()
```

Comment:
admittedly the diffs are a bit confusing; can you explain where this code went, or whether it was really deleted?

Reply:
The value of `self.nonce` is not used anywhere in this module or by any other module which interacts this module.

Reply:
that was just the beginning of a code block of a few dozen lines that you deleted and that was doing stuff. same qn as above -- how confident are you that this code really was not needed?

Reply:
The code in the two internal functions - `_get_transaction_info` and `_get_prices_info` was moved into the only function which calls them (one by one) - `validate_and_submit_transaction`, which will later probably go through some additional renovation until its flow is clear and easy to follow.